src/components/ScholarshipAmountField.jsx

```diff
@@ -35,45 +35,42 @@ function ScholarshipAmountField(props) {
     updateAmount,
   } = props;
 
-  const isRange = amountType === AmountType.Range;
-  const isFixed = amountType === AmountType.Fixed;
   const error = !!helperText; // no error if helperText empty
 
   const inputFields = {};
-  inputFields[AmountType.Range] = (
+  inputFields[AmountType.Varies] = (
     <>
       <AmountTextField
-        error={isRange && error && (!!maxAmount || minAmount >= maxAmount)}
+        error={
+          error && (minAmount < 0 || (minAmount && minAmount >= maxAmount))
```

Comment:
`minAmount < 0` is not possible per constraints in [AmountTextField](https://github.com/beyondhb1079/s4us/blob/main/src/components/AmountTextField.jsx#L23)
```suggestion
          error && (minAmount && minAmount >= maxAmount)
```

Reply:
I will pass the formik object to AmountTextField so we can directly know what field has the error.